### DISCUSSION TITLE
"GitHub"-like diff, with highlighted diff

### DISCUSSION THREAD

**Discussion by GuillaumeDesforges** [marp-team/marp#344] [Q&A] [answered]
I can render a diff using

~~~markdown
```diff
- Hello world!
+ Hello, world!
```
~~~

renders:

```diff
- Hello world!
+ Hello, world!
```

Now, is it possible with `marp` for the diffed part on the line to be highlighted, like on GitHub PRs?

**Comment by yhatt** [comment, marked as answer]
Marp Core is using [highlight.js](https://github.com/highlightjs/highlight.js) to render highlights. `diff` language is always highlighting per lines.

The language for rendering word-level diffs on the same line had been suggested at https://github.com/highlightjs/highlight.js/issues/2911 as `worddiff` language. At least I could not find out the 3rd-party highlight.js plugin of `worddiff` language for now, so I guess currently there is no way to get word-level diff highlights. (If there was a highlight.js plugin, you would be able to apply it through Marp Core highlighter overloading via Marp plugin)

**Reply by GuillaumeDesforges** [reply]
Thanks for the precise pointers.

I'll mark as answer for now then, since it's now up to me (or a benevolent spirit) to make it happen.
